internal/pkg/service/common/etcdop/iterator/iterator_typed.go

```diff
@@ -11,14 +11,22 @@ import (
 )
 
 type DefinitionT[T any] struct {
-	config
-	filters []func(*op.KeyValueT[T]) bool // true means accepting the value
+	Definition
+	// serde is serialization/deserialization implementation, it is empty for not-typed iterator
+	serde *serde.Serde
+	// filters - true means accepting the value
```

Comment:
`serde`: moved from the common `config` for typed/non-typed iterator to typed `DefinitionT/IteratorT`.

Reply:
Suport of new features is also implemented in the `DefinitionT/IteratorT` because they embed untypped `Definition/Iterator`.